oeps/processes/oep-0021-proc-deprecation.rst

```diff
@@ -268,6 +311,8 @@ guidance. In some cases, the proposal may need to be *Abandoned* entirely.
 
 .. _edX Architecture Team: https://openedx.atlassian.net/wiki/spaces/AC/pages/439353453/Architecture+Team
 
+**For Code Review: This link is not publicly available. Where should we direct people?**
```

Comment:
Code reviewers: Please advise

Reply:
Oh wow, that's really old. I would maybe suggest the deprecation working group page (which I believe should be public): https://openedx.atlassian.net/wiki/spaces/AC/pages/825983190/Deprecation+Working+Group